gst/tensor_filter/tensor_filter_tensorflow_lite_core.cc

```diff
@@ -103,6 +103,18 @@ TFLiteCore::loadModel ()
       _print_log ("Failed to construct interpreter\n");
       return -2;
     }
+
+    if (interpreter->AllocateTensors () != kTfLiteOk) {
+      printf ("Failed to allocate tensors\n");
```

Comment:
please change to  '_print_log',  not 'printf'

Reply:
I think that input-tensor also would be set to dynamic allocation.
Please review the changes I sent.